src/features/comments.ts

```diff
@@ -0,0 +1,80 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Comment, CommentData } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+export const fetchComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => getPostComments(postId),
+);
+
+export const addComment = createAsyncThunk(
+  'comments/add',
+  (commentData: CommentData & { postId: number }) => createComment(commentData),
+);
+
+export const deleteComments = createAsyncThunk(
+  'comments/delete',
+  (commentId: number) => {
+    deleteComment(commentId);
```

Comment:
The `deleteComment` function should return a promise. Consider returning the result of `deleteComment(commentId)` to ensure that the thunk handles the asynchronous operation correctly.